core/platform/models.py

```diff
@@ -27,8 +27,17 @@
 
 
 class _Platform(object):
+    """Acts as a base class for platform-specific imports related to GAE
+    """
     @classmethod
     def import_models(cls):
+        """Acts as an abstract method that should be implemented on inherited
+            classes.
```

Comment:
Remove indentation in the second line. Only `Args:` and `Returns:` must have continuation lines indented by 4 spaces.